pkg/license/download.go

```diff
@@ -138,54 +139,42 @@ func (ddi *DefaultDownloaderImpl) SetOptions(opts *DownloaderOptions) {
 	ddi.Options = opts
 }
 
+//go:embed licenseData
+var licenseContent embed.FS
+
 // GetLicenses downloads the main json file listing all SPDX supported licenses
 func (ddi *DefaultDownloaderImpl) GetLicenses() (licenses *List, err error) {
```

Comment:
Here, I think we want to keep the code to download the licenses, at some point we may need to let users generate an SBOM with license version 1.xxx which will not be embedded and then we will need to download it.

Reply:
Agreed. 

Right now this function only does deserialization. We would need to rename/restructure the code here.

Reply:
Ok upon a quick look to embed FS docs, the embed.FS is read only. Hence we won' be able to  allow users to refresh their licenses. The only way to get version X of license list would be to download the binary of `bom` which embeds it ):